frontend/src/components/static_pages/about.css

```diff
@@ -35,6 +41,10 @@
   .heading {
     font-size: 30px;
   }
+  .people-heading {
+    font-size: 30px;
+    font-weight: bold;
```

Comment:
this the same as the style without the screen check, so it is unneeded. 

Only need to add styles in here that are different like how the font size is changing

Reply:
On hold until further details on how to treat the photos.